packages/financial-templates-lib/src/price-feed/UniswapPriceFeed.js

```diff
@@ -92,18 +92,45 @@ class UniswapPriceFeed extends PriceFeedInterface {
     // Approximate the first block from which we'll need price data from based on the
     // lookback and twap length:
     const lookbackWindow = this.twapLength + this.historicalLookback;
+    const currentTime = await this.getTime();
```

Comment:
Can you assume that getTime is async?

Reply:
We cannot, but `await` just passes the value through if it isn't a promise: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await, so no special casing should be necessary.